Formula/libsql-server.rb

```diff
@@ -0,0 +1,26 @@
+class LibsqlServer < Formula
+  desc "Server-mode binary for libSQL (sqld), an Open Contribution fork of SQLite"
+  homepage "https://libsql.org"
+  url "https://github.com/libsql/sqld/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "d3b222ed54c4ab1707da16198760f8619d5cdbfbc6ecd72998671100ddc85439"
+  license "MIT"
+
+  depends_on "rust" => :build
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "rust" => :build
```
This should fix your Linux build failure.